.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,10 +18,7 @@
 - [ ] My name is on the [list of contributors](https://github.com/glotzerlab/signac/blob/master/contributors.yaml).
 - [ ] My code follows the [code style guideline](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md#code-style) of this project.
 - [ ] The changes introduced by this pull request are covered by existing or newly introduced tests.
-
-If necessary:
-- [ ] I have updated the API documentation as part of the package doc-strings.
-- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
+- [ ] I have updated the API documentation as part of the package doc-strings, or this is not necessary.
+- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here, or this is not necessary.
```

Comment:
```suggestion
- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here. / No change to the framework documentation needed (see below for guidance).
```

Here is a first _crude_ attempt at this guidance:

> #### Guidance on [framework documentation](https://docs.signac.io) updates:
>
> - Any change that affects user-facing features (labeled `enhancement`) must be documented.
> - Breaking changes (labeled `breaking`) may require a revision.
> - Changes that are only affecting purely internal components do not require an update.

Reply:
I've revised these guidelines and made the checklist item start by determining if an update is needed.